### PR TITLE
M3-5901: Support Pro Dedicated Plans

### PR DESCRIPTION
**Pull request by HanaXu**
## Description
Add support for `prodedicated` plan types in the Linode Create flow

image

## How to test
Go to `/linodes/create` while pointing to dev or turn on the mocks

### CODE REVIEW

File: packages/manager/src/features/linodes/LinodesCreate/SelectPlanPanel.tsx [hunk 1]
```diff
@@ -410,6 +414,24 @@ export const SelectPlanPanel: React.FC<CombinedProps> = (props) => {
...
 
+    if (!isEmpty(proDedicated)) {
+      tabs.push({
+        render: () => {
+          return (
+            <>
+              <Typography data-qa-prodedi className={classes.copy}>
+                Pro Dedicated CPU instances are for very large plans. They only
+                have AMD 2nd generation processors or above.
+              </Typography>
```
Comment (bnussman) [inline packages/manager/src/features/linodes/LinodesCreate/SelectPlanPanel.tsx:425]: Is this copy temporary? It does not sound right to me. I left what I think sounds better below.

```suggestion
              <Typography data-qa-prodedi className={classes.copy}>
                Pro Dedicated CPU instances are for very demanding workloads. They
                have AMD Epyc 2nd generation processors or newer.
              </Typography>
```

**Review by bnussman — CHANGES_REQUESTED** [review]
I don't think `Pro Dedicated CPU` should be the first item in the plans table. Can it go after `Dedicated CPU`? If a customer saw a $960 monthly price as the first plan we show, they may be very disturbed.

File: packages/manager/src/features/linodes/LinodesCreate/SelectPlanPanel.tsx (hunk 1 above)
Comment (HanaXu) [reply packages/manager/src/features/linodes/LinodesCreate/SelectPlanPanel.tsx:425]: Not sure if `AMD Epyc` has been finalized so just going to leave it as `AMD 2nd generation` for now

**Review by DevDW — APPROVED** [review]
LGTM assuming the copy has been ran by stakeholders (even if it's still subject to change at the moment)

**Review by jdamore-linode — APPROVED** [review]
Looks good!

**Pull request merged**